src/core/event/vlogger_timer_handler.cpp

```diff
@@ -65,7 +65,4 @@ void vlogger_timer_handler::handle_timer_expired(void *user_data)
     if (g_p_vlogger_level) {
         g_vlogger_level = *g_p_vlogger_level;
     }
-    if (g_p_vlogger_details) {
```

Comment:
can you explain? thanks

Reply:
in classic xlio - we could control the details (whether to print cpp file, line no, etc.)

in doca - there's no details control so anything related to details was deleted.